api/v1alpha1/watermarkpodautoscaler_types.go

```diff
@@ -204,3 +204,17 @@ type WatermarkPodAutoscalerList struct {
 func init() {
 	SchemeBuilder.Register(&WatermarkPodAutoscaler{}, &WatermarkPodAutoscalerList{})
 }
+
+// GetLogAttrs takes a list of log attrs that would be sent to logger.info, and adds all key value label pairs from
+// the WPA spec. logAttrs should be pairs of key/values, with each key being a string.
+func (wpa *WatermarkPodAutoscaler) GetLogAttrs(logAttrs ...interface{}) []interface{} {
+	var logAttributes []interface{}
+	for k,v := range wpa.ObjectMeta.Labels {
```

Comment:
I think we should have a whitelist of labels that we want to add, that could be configurable